### DISCUSSION TITLE
Identifizieren Puffer

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#211] [General]
Hallo Jürgen,
beim Tool "Identifizieren"
![image](https://github.com/user-attachments/assets/1a7aabaa-c603-4c94-a02e-03eb74e952f8)
wird der Puffer-Bereich in der Karte nicht angezeigt.
lg

**Comment by gis-eni** [comment]
Der Puffer wird erst sichtbar, wenn man auf ``Anwenden`` klickt:

<img width="749" alt="image" src="https://github.com/user-attachments/assets/d82d279a-2d28-4b10-8908-385da4c1a414">

Erst dann wird auch die Abfrage durchgeführt. Der Screenshot kommt aus dem aktuellen **Atlas Steiermark**. Das gleiche Verhalten hab auch ich bei mir... as designed

**Comment by marko-jer** [comment]
noch ein paar Infos damit es nachvollziehbarerer ist:

Digitaler Atlas Steiermark:![image](https://github.com/user-attachments/assets/6e603f72-57d4-4fa7-8b70-08c4f3c82dc3)

Der Fehler tritt aber nur in der Version 6.x auf und nur im Layout _Basic/Mobile_:
intern:![image](https://github.com/user-attachments/assets/353384e3-8106-4fbe-9eae-1699d53e736c)
Layout: Basic/Mobile

Ausgangsparameter:
![image](https://github.com/user-attachments/assets/a6ed5002-aca7-4a54-bbe5-32bae47784a4)
nach dem Anwenden:
![image](https://github.com/user-attachments/assets/61343ef0-3949-4fa0-87a7-e7f5e4eb8b32)

Conclusio:
der Puffer wird gar nicht angewandt, nicht nur nicht angezeigt!

Ich hoffe, jetzt ist es dann nachvollziehbar.

**Comment by gis-eni** [comment]
Ok, dass kann ich nachvollziehen. Ist mit Build 6.24.4402 behoben
